cli/silence_import.go

```diff
@@ -103,7 +102,7 @@ func (c *silenceImportCmd) bulkImport(ctx context.Context, _ *kingpin.ParseConte
 		wg.Add(1)
 		go func() {
 			addSilenceWorker(ctx, amclient.Silence, silencec, errc)
-			wg.Done()
+			defer wg.Done()
```

Comment:
What's the use of deferring here? This is the last thing in the function. It will be executed immediately anyway.